drivers/unix/os_unix.h

```diff
@@ -48,6 +48,7 @@ class OS_Unix : public OS {
 
 public:
 	OS_Unix();
+	virtual ~OS_Unix();
```

Comment:
Is this needed? `OS` already has a virtual destructor

Reply:
Perhaps not, though I recall that when I was breakpointing through this, the stack traces looked a bit different - and clearer in my opinion - when I explicitly declared this intermediately inherited virtual destructor.